avsomr/avslog.h

```diff
@@ -0,0 +1,24 @@
+#ifndef AVS_LOG_H
+#define AVS_LOG_H
+
+#include <QDebug>
```

Comment:
not QtDebug? That is in all.h though

Reply:
<QDebug> and <QtDebug> this is the same - #include "qdebug.h"